File: MODUL04/PraktikumPy/2.py
def urutan(n):
    for i in range(1, n + 1, 2):
        print(i, end=" ")
    print()
    if n % 2 == 1:
        n -= 1

    for i in range(n, 1, -2):
        print(i, end=" ")

if __name__ == "__main__":
    max = int(input(""))
    urutan(max)

File: MODUL04/PraktukumC/2.c
#include <stdio.h>

void urutan(int n) {
    for (int i = 1; i <= n; i += 2) {
        printf("%d ", i);
    }
    printf("\n");
    
    if (n % 2 == 1) {
        n--;
    }
    for (int i = n; i >= 2; i -= 2) {
        printf("%d ", i);
    }
}

int main() {
    int max;
    scanf("%d", &max);
    urutan(max);

    return 0;
}
